docker/ccf-pdo.sgx.yaml

```diff
@@ -0,0 +1,51 @@
+# Copyright 2017 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
```

Comment:
this is too much of a copy/paste. fix the copyright date. fix the comments (below). for example, this file doesn't really do anything with sawtooth so why are we referencing it in the comments.

Reply:
thank you, will fix the Copyright and the header.